Sources/SwiftKuery/ConnectionPoolConnection.swift

```diff
@@ -145,23 +180,25 @@ public class ConnectionPoolConnection: Connection {
         connection.execute(raw, parameters: parameters, onCompletion: onCompletion)
     }
     
-    /// Prepare statement.
+    // MARK: Prepared statements
+
+    /// Create a prepared statement from the passed in query.
     ///
-    /// - Parameter query: The query to prepare statement for.
+    /// - Parameter query: The query to prepare the statement for.
     /// - Returns: The prepared statement.
-    /// - Throws: QueryError.syntaxError if query build fails, or a database error if it fails to prepare statement.
+    /// - Throws: QueryError.syntaxError if the query build fails, or a database error if it fails to prepare statement.
     public func prepareStatement(_ query: Query) throws -> PreparedStatement {
         guard let connection = connection else {
             throw QueryError.connection("Connection is disconnected")
         }
         return try connection.prepareStatement(query)
     }
     
-    /// Prepare statement.
+    /// Create a prepared statement from the passed in query.
     ///
-    /// - Parameter raw: A String with the query to prepare statement for.
+    /// - Parameter raw: A String with the query to the prepare statement for.
```

Comment:
"A String with the query to the prepare statement for." - the is in the wrong place, it should be
"A String containing the query to prepare the statement for."